libs/contrib/Text/Parser/Core.idr

```diff
@@ -271,6 +271,14 @@ position = Position
 public export
 data ParsingError tok = Error String (Maybe Bounds)
 
+public export
```

Comment:
Does it need to be `public`? I don't think we'll run this at the type level.

Reply:
Probably not, in that case. I tend to use `public` as a "this implementation is not secret" thing, but if it's detrimental, then it can be removed  : )

Reply:
Changed  : )